acceptance/bundle/deployment/bind/schema/databricks.yml

```diff
@@ -0,0 +1,10 @@
+bundle:
+  name: bind-schema-test-$BUNDLE_UUID
```

Comment:
$BUNDLE_UUID can be confused with bundle.uuid, so I'd use different name.

Reply:
fixed in a follow-up commit